kolibri_explore_plugin/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = "6.21.0"
+from ._version import __version__  # noqa: F401
```

Comment:
Do you know why this is still needed?

Reply:
It’s needed by `webpack_json.py` (part of kolibri-tools), as it tries to dereference the `__version__` member of every plugin module it loads. It’s caught by the CI when doing the kolibri-tools steps of the build, but I didn’t catch it locally earlier because the dereference is dynamic, and comes from the kolibri-tools module rather than kolibri-explore-plugin, so doesn’t show up in `git grep` or linting.

I’ll amend the commit message to give details of that, when I next update the branch.